deployment_scripts/linode-marketplace-nats-single-node/nats-single-node-deploy.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+set -e
+trap "cleanup $? $LINENO" EXIT
+
+##Linode/SSH security settings
+#<UDF name="user_name" label="The limited sudo user to be created for the Linode" default="">
+#<UDF name="password" label="The password for the limited sudo user" example="an0th3r_s3cure_p4ssw0rd" default="">
+#<UDF name="disable_root" label="Disable root access over SSH?" oneOf="Yes,No" default="No">
+#<UDF name="pubkey" label="The SSH Public Key that will be used to access the Linode (Recommended)" default="">
+
+## Domain Settings
+#<UDF name="token_password" label="Your Linode API token. This is needed to create your server's DNS records" default="">
+#<UDF name="subdomain" label="Subdomain" example="The subdomain for the DNS record: www (Requires Domain)" default="">
+#<UDF name="domain" label="Domain" example="The domain for the DNS record: example.com (Requires API token)" default="">
+
+## nats Settings 
+# <UDF name="name" label="NATS Server Name" default="Test" />
+# <UDF name="version" label="Version" oneOf="2.10.1,2.10.0,2.9.22,2.9.21,2.9.20" default="2.10.1" />
+# <UDF name="system_user_password" label="System User Password" />
+# <UDF name="example_user_password" label="Example User Password" />
```

Comment:
Can we generate these passwords in a provision.yml to match the style of the other playbooks

Reply:
Done!